RunningOrder/Sprint/SprintList.swift

```diff
@@ -17,25 +17,39 @@ struct SprintList: View {
     @State private var showNewSprintModal = false
 
     var body: some View {
-        List {
-            Section(header: Text("Active Sprints")) {
-                ForEach(sprints) { sprint in
-                    NavigationLink(
-                        destination: StoryList(stories: sprint.stories)
-                            .listStyle(PlainListStyle()),
-                        label: {
-                            Text(sprint.name)
-                        })
+        VStack(alignment: .leading) {
+            List {
+                Section(header: Text("Active Sprints")) {
+                    ForEach(sprints) { sprint in
+                        NavigationLink(
+                            destination: StoryList(stories: sprint.stories)
+                                .listStyle(PlainListStyle()),
+                            label: {
+                                HStack {
+                                    SprintNumber(number: sprint.number, color: sprint.color)
+                                    Text(sprint.name)
+                                }
+                            })
+                    }
+                }
+                Section(header: Text("Old Sprints")) {
+                    EmptyView()
                 }
             }
 
             Button(action: { self.showNewSprintModal.toggle() }) {
-                Text("New Sprint")
-            }
-            Section(header: Text("Old Sprints")) {
-                EmptyView()
+                HStack {
+                    Image(nsImage: NSImage(named: NSImage.addTemplateName)!)
+                        .frame(width: 20, height: 20)
```

Comment:
> maybe with resizable() modifier

> Why Resizable on the image ? The cross would fill all the circle

yes, but when you have to resize the image as you do with frame, I thought you needed to use resizable